### PR TITLE
report fixes

### PR DESCRIPTION
**Pull request by binaryberserker**
TR-151: Improve toast messaging
TR-154: Saved report names and sort order

image
image

### CODE REVIEW

File: public/css/primitives.css [hunk 1]
```diff
@@ -7,8 +7,8 @@
 	--last-period-warning-color: #c78a27;
 	--unknown-color: #dab092;
 	--notstarted-color: #97A0AF; /* N80 - Spooky ghost */
-	--blocked-color: rgb(242, 0, 11);
-	--last-period-blocked-color: rgb(176, 40, 47);
+	--blocked-color: #f2000b;
+	--last-period-blocked-color: #b0282f;
```
Comment (binaryberserker) [inline public/css/primitives.css:11]: this is the same color but in hex, for some reason vscode wasn't showing me the color with RGB like it does with hex, can change it back if needed

File: public/css/primitives.css [hunk 2]
```diff
@@ -26,6 +26,15 @@
 	--bitovi-red-orange: #F5532D;
 
 
+	/* design system, for Atlassian */
+	--ds-icon-success: var(--complete-color);
+	--ds-icon-warning: var(--warning-color);
+	--ds-icon-danger: var(--blocked-color);
+	--ds-text-success: #36885e;
+	--ds-text-warning: var(--last-period-warning-color);
+	--ds-text-danger: var(--last-period-blocked-color);
```
Comment (binaryberserker) [inline public/css/primitives.css:35]: atlaskit needs these vars defined in order to colors in the font or icons

File: public/react/ViewReports/ViewReports.tsx [hunk 3]
```diff
@@ -41,7 +42,7 @@ const ViewReports: FC<ViewReportProps> = ({ onBackButtonClicked }) => {
                 href={"?" + report.queryParams}
                 className="flex items-center font-normal text-sm leading-5 h-10"
               >
-                Report name {report.name}
+                {report.name}
```
Comment (binaryberserker) [inline public/react/ViewReports/ViewReports.tsx:45]: removed "Report name"

File: public/shared/messaging.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import SuccessIcon from "@atlaskit/icon/core/success";
+import ErrorIcon from "@atlaskit/icon/core/error";
+import { Text } from "@atlaskit/primitives";
+import { token } from "@atlaskit/tokens";
+
+export const errorIcon = <ErrorIcon color={token("color.icon.danger")} label="error" />;
+export const successIcon = <SuccessIcon color={token("color.icon.success")} label="success" />;
+
+export const errorHeader = (msg = "Uh Oh!") => <Text color="color.text.danger">{msg}</Text>;
+export const successHeader = (msg = "Success") => <Text color="color.text.success">{msg}</Text>;
```
Comment (binaryberserker) [inline public/shared/messaging.tsx:11]: using these icons and header messages for the team config and report toast messages

File: public/react/SaveReports/components/SavedReportDropdown/RecentReports.tsx [hunk 5]
```diff
@@ -19,12 +20,21 @@ const RecentReports: FC<RecentReportsProps> = ({
   onViewReportsButtonClicked,
   setIsOpen,
 }) => {
+  const reportsToShow = useMemo(
+    () =>
+      recentReports
+        .map((id) => reports[id])
+        .filter(notEmpty)
+        .sort((a, b) => a.name.localeCompare(b.name)),
```
[the remark this answers is not in the document]
Comment (justinbmeyer) [reply public/react/SaveReports/components/SavedReportDropdown/RecentReports.tsx:28]: does this ignore case?

File: public/shared/messaging.tsx (hunk 4 above)
Comment (DavidNic11) [reply public/shared/messaging.tsx:11]: These are components, we need to treat them as such

**Review by DavidNic11 — CHANGES_REQUESTED** [review]

File: public/react/SaveReports/components/SavedReportDropdown/RecentReports.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (binaryberserker) [reply public/react/SaveReports/components/SavedReportDropdown/RecentReports.tsx:28]: yes

**Review by DavidNic11 — APPROVED** [review]

**Pull request merged**